### DISCUSSION TITLE
PatchTool 6.25 10/12

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#236] [Announcements]
僅做新版本的相容性更新，順便更新了幾句技能更動。

目前漢化至中國服6.15，其餘只有零星漢化。

檔案：
- FFXIVChnTextPatch-20221018.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：
- rawexd-zhcn-20221018.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- backup-20221018.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

目前測試，在之前的更新後，第三方4k字體包可能會導致遊戲崩潰。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.70.1'>PatchTool 6.25 10/12</a>.</em>

**Comment by okami2223** [comment]
你好，在漢化版本86級主線需要在一般頻道說話
但輸入文字不是英文，而是中文的"我有個請求"
![螢幕擷取畫面 2022-10-31 105542](https://user-images.githubusercontent.com/44223291/198921634-62061f77-a251-45a7-8021-4a6b912df2cc.png)

[再麻煩作者協助修正，辛苦了！

**Comment by Bos-hunting** [comment]
希望有一個6.28 的暫時兼容版 麻煩了謝謝
